src/Card-Component.jsx

```diff
@@ -0,0 +1,28 @@
+import { UseGame } from "./game-context";
+
+import {BsFillQuestionCircleFill} from 'react-icons/bs';
+const CartComponent = (props) => {
+  const { flipCard, setFlipCard, setTwoCounter } = UseGame();
+  const clickHandler = () => {
+    setTwoCounter((twoCounter) => twoCounter + 1);
+    setFlipCard(
+      flipCard.map((obj) =>
+        obj.letter === props.ltr
+          ? { ...obj, tempshow: !obj.tempshow, clicked: !obj.clicked }
+          : { ...obj }
+      )
+    );
+  };
+  const letterObj = flipCard.find((obj) => obj.letter === props.ltr);
+  
+
+  return (
+    <div onClick={() => clickHandler()}>
```

Comment:
clickHandler can be named for the exact function it is performing, would make it more readable

Reply:
Changed as per suggestion.